enhancements/cluster-scope-secret-volumes/shared-resources-via-openshift-builds-operator.md

```diff
@@ -0,0 +1,291 @@
+---
+title: shared-resources-via-openshift-builds-operator
+authors:
+  - "@adambkaplan"
+  - "@gabemontero"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@cdaley"
+  - "@rgormley"
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - "@bparees"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@bparees"
+creation-date: 2023-08-17
+last-updated: 2023-08-22
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/RHDP-701
+see-also:
+  - "enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md"
+  - "enhancements/cluster-scope-secret-volumes/shared-resource-validation.md"
+replaces:
+  - "/enhancements/subscription-content/subscription-injection.md"
+superseded-by: []
+---
+
+# Shared Resources via OpenShift Builds Operator
+
+## Summary
+
+Remove the Shared Resources CSI driver from OpenShift, and deliver it to customers with the
+forthcoming OpenShift Builds OLM Operator.
+
+
+## Motivation
+
+The Shared Resources CSI driver provides a novel way for sharing information across OpenShift
+namespaces. Though there was hope this capability would be generally useful, in practice this
+feature is mainly needed for Red Hat build workloads.
+
+Packaging this driver with the forthcoming OpenShift Builds operator provides several advantages:
+
+- Provides an extension to OpenShift Builds, which is based on the upstream [Shipwright](https://shipwright.io) project.
+- Allows release outside of OpenShift's cadence.
+- Potential to fully support customers on older versions of OCP.
+- Pathway to adoption on hosted control plane clusters.
```

Comment:
This seems to be answered in Alternatives -> Remain in OCP section, I think it would be useful to elaborate this issue in the motivation (is it a main motivation of this change actually?)

Reply:
I would agree @petr-muller.   This was a key motivation in fact. WDYT @adambkaplan ?

Reply:
Updated motivation